packages/netlify-cms-core/index.d.ts

```diff
@@ -403,6 +404,13 @@ declare module 'netlify-cms-core' {
     config?: CmsMediaLibraryOptions;
   }
 
+  export interface CmsEventListener {
+    name: string;
```

Comment:
Should we limit the `name` the to list of allowed events?
https://github.com/netlify/netlify-cms/blob/567438fdd1aa775fd09a6a5de13e79c9d783e0d6/packages/netlify-cms-core/src/lib/registry.js#L6